loadgen/loadgen.cc

```diff
@@ -802,15 +837,19 @@ void PerformanceSummary::Log(AsyncLog& log) {
       break;
     }
     case TestScenario::Offline: {
-      double qps = sample_count / pr.max_latency;
-      log.LogSummary("QPS: ", qps);
+      double samples_per_second = sample_count / pr.max_latency;
+      log.LogSummary("Samples per second: ", samples_per_second);
```

Comment:
@christ1ne : This is where the summary log now refers to "samples per second" rather than "QPS".

Reply:
thanks! I missed that earlier.